pkg/controller/installation/installation_controller.go

```diff
@@ -572,6 +573,33 @@ func (r *ReconcileInstallation) processStage(installation *integreatlyv1alpha1.R
 	return integreatlyv1alpha1.PhaseCompleted, mErr
 }
 
+// handle the deletion of CRO config map
+func (r *ReconcileInstallation) handleCROConfigDeletion(rhmi integreatlyv1alpha1.RHMI) error {
+	// get cloud resource config map
+	croConf := &corev1.ConfigMap{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: rhmi.Namespace, Name: "cloud-resource-config"}, croConf)
```

Comment:
I was thinking maybe we should have a single file containing exported constants that would be used across different files so that:
* we wouldn't need to make sure the names are correct when this constant is used in multiple places
* we would have one place to change if we ever change the name of this resource

Just a thought since `cloud-resource-config` is also defined in the `bootStrapReconciler.go`
https://github.com/integr8ly/integreatly-operator/blob/3bcd0e6b8a818d522a35f2f6960e449d029ae715/pkg/controller/installation/bootstrapReconciler.go#L91

Not too sure on people's thoughts on this though :thinking: